test/api_compatibility_helpers.js

```diff
@@ -0,0 +1,44 @@
+var _      = require('underscore');
+
+exports.hasSpanFunctions =
+function hasSpanFunctions(span) {
+    var funcs = [
+        'setBaggageItem',
+        'getBaggageItem',
+        'context',
+        'tracer',
+        'setOperationName',
+        'finish',
+        'addTags',
+        'log',
+        'logEvent',
+        'setTag'
+    ];
+    return _.every(funcs, function(name) {
```

Comment:
I would rather pass the `assert` object to this function and use `isOK` method, so that if a particular function is missing the test output would explicitly list it.